thoth/storages/data/alembic/versions/930b47e27b6c_rename_thoth_s2i_columns.py

```diff
@@ -0,0 +1,36 @@
+"""rename thoth s2i columns
+
+Revision ID: 930b47e27b6c
+Revises: c427c501f966
+Create Date: 2022-01-19 16:56:54.439445+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "930b47e27b6c"
+down_revision = "c427c501f966"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column("external_software_enviroment", "thoth_s2i_image_name", new_column_name="thoth_image_name")
```

Comment:
typo in the name of the columns, here an below: `external_software_environment` and `software_environment`

Reply:
Thank you, I'll correct that right away :+1: